src/server/index.jsx

```diff
@@ -166,10 +174,13 @@ server.get(
         getAgent,
       });
 
+      const mvtExperiments = getMvtExperiments(headersTest);
+
       data.toggles = toggles;
       data.path = urlPath;
       data.timeOnServer = Date.now();
-      data.showAdsBasedOnLocation = headers['bbc-adverts'] === 'true';
+      data.showAdsBasedOnLocation = headersTest['bbc-adverts'] === 'true';
+      data.mvtExperiments = mvtExperiments;
```

Comment:
Currently, this will pass all experiments we derive from the belfrage headers, regardless of whether the experiment is enabled, thinking of changing this to be less of a headache on the front for having to check what service/page type it will be shown on. See this comment https://github.com/bbc/simorgh/pull/10208#discussion_r932346604